source/dub/dub.d

```diff
@@ -655,7 +678,7 @@ class Dub {
 		}
 
 		if (existing) {
-			if (!ver.startsWith("~") || !(options & FetchOptions.forceBranchUpgrade) || location == PlacementLocation.local) {
+			if (!ver.startsWith("~") || !(options & FetchOptions.forceBranchUpgrade)/* || location == PlacementLocation.local*/) {
```

Comment:
@s-ludwig This was introduced in https://github.com/dlang/dub/commit/2be278a19b74a5791886bfe8963551d865f27e0a, perhaps you could talk me through it?